doc/tasks/securitychecker/composeraudit.md

```diff
@@ -0,0 +1,48 @@
+# Composer Audit Security Checker
+
+The Security Checker will check your `composer.lock` file for known security vulnerabilities.
+
+***Config***
+
+The task lives under the `securitychecker_composeraudit` namespace and has the following configurable parameters:
+
+```yaml
+# grumphp.yml
+grumphp:
+    tasks:
+        securitychecker_composeraudit:
+            locked: true
+            no_dev: false
+            run_always: false
+            working_dir: ./
+```
+
+**format**
```

Comment:
Can you add this one (with default value) to the code sample as well - for completeness.